index.md

```diff
@@ -43,9 +43,9 @@ like this Python code example:
 import os, sys
 
 def has_colors():
-    if os.environ.get('NO_COLOR'):
+    if os.environ.get('NO_COLOR', '') != '':
```

Comment:
citing https://no-color.org/

> NO_COLOR environment variable that, when present and not an empty string (regardless of its value),

```
% unset NO_COLOR ; python3 -c "import os; print(bool(os.environ.get('NO_COLOR')))"
False
% NO_COLOR= python3 -c "import os; print(bool(os.environ.get('NO_COLOR')))"
False
% NO_COLOR=1 python3 -c "import os; print(bool(os.environ.get('NO_COLOR')))"
True
```

so the previous code follows the spec without this modification

plus this modification makes it more confusing and less idiomatic

Reply:
Neat. I agree, though the linked citation does not include any python code. I'll close this PR.